docs/lib/duckdb.md

```diff
@@ -105,3 +105,13 @@ const sql = DuckDBClient.sql({quakes: `https://earthquake.usgs.gov/earthquakes/f
 ```sql echo
 SELECT * FROM quakes ORDER BY updated DESC;
 ```
+
+## Extensions
+
+DuckDB’s [extensions](../sql#extensions)<a href="https://github.com/observablehq/framework/pull/1734" class="observablehq-version-badge" data-version="prerelease" title="Added in #1734"></a> are supported.
+
+By default, `DuckDBClient.of` and `DuckDBClient.sql` load the (self-hosted) extensions referenced in the [configuration](../config#duckdb). You can pass an options object with a **load** key set to a different list if you want a different environment:
+
+```js echo run=false
+const geodb = await DuckDBClient.of({}, {load: ["spatial", "h3"]});
```

Comment:
Are you only able to load extensions that were previously registered in the config? Or would it be possible to register a “one-off” extension here? And presumably you’d need to use `import.meta.resolve` to self-host the URL to the extension.

Reply:
This would be great. Maybe we shouldn't expose this "load" option until we manage to implement it as you're suggesting?